Djikstra/Djikstra/Functions.cpp

```diff
@@ -0,0 +1,135 @@
+#include <vector>
+
+
+#include "Headers.h"
+
+
+namespace Graph
+{
+	bool Graph::node_exist( std::string name)
+	{
+		int i=0, vector_size;
+		bool exist = 0;
+		vector_size = m_vertex.size();
+		
+		while ( i < vector_size )
+		{
+			if (name ==  m_vertex[i].name ) //if name exist break loop, and function
+			{
+				exist = 1;
+				break;
+			}
+			i++;
+		}
+
+		return exist; //if vertex exist, then return 1;
+	}
+
+
+
+	void Graph::add_node( std::string name)
+	{
+		int vector_size;
+
+		vector_size = m_vertex.size();
```

Comment:
You don't need this variable each time. You can directly use the result of a call `m_vertex.size()`.